### DISCUSSION TITLE
Migrating away from a deprecated version of the Sass compiler

### DISCUSSION THREAD

**Discussion by lucadallavalle** [bootstrap-email/bootstrap-email#151] [Ideas]
Hi,
I have some problems in compiling a project including external scss files and the cause seems to lie on the fact that bootstrap-email makes use of `sassc`, which is a wrapper around `LibSass`, both of which are deprecated (and the newest Sass features are not available).
I tried to look for an issue documenting this problem but I couldn't find it. Were you aware of the situation? Are there any plans in this regard?
Thank you for your project.

More info: https://sass-lang.com/blog/libsass-is-deprecated

**Comment by stuyam** [comment]
That's for bringing this to my attention. I feels like just yesterday I migrated from Ruby Sass to SassC and now SassC is Deprecated 😂  I had no idea. Honestly Im not 100% what the options are right now. It isn't clear if there is a ruby wrapper for DartSass yet or if there will every be one. If anyone knows what options there are out there please let me know Im open to ideas.
